cmd/podman/containers/utils_linux.go

```diff
@@ -0,0 +1,15 @@
+package containers
+
+import (
+	"syscall"
+
+	"github.com/pkg/errors"
+	"golang.org/x/sys/unix"
+)
+
+func verifyFD(fd int) error {
+	if _, err := unix.FcntlInt(uintptr(fd), unix.F_GETFL, 0); err != nil && err != syscall.Errno(0) {
+		return errors.Wrapf(err, "failed to verify %d file descriptor", fd)
+	}
```

Comment:
this won't detect fds that are opened by Podman.

Reply:
If run `podman run --preserve-fds`, it's possible that the file descriptor I want to pass has already opened by podman, so this won't detect that?